### DISCUSSION TITLE
Drafting the Kubebuilder 2025 Roadmap – Share Your Feedback! 🚀

### DISCUSSION THREAD

**Discussion by camilamacedo86** [kubernetes-sigs/kubebuilder#4387] [Announcements]
Hey Everyone,

We've started drafting our primary goals and priorities for 2025, and we're excited to share them with you! The roadmap is now available in the roadmaps directory, aiming to enhance transparency, clarify our direction, and keep us focused on key priorities for the year ahead.

Check out the PR: https://github.com/kubernetes-sigs/kubebuilder/pull/4316

The current draft outlines three main goals, and we’d love to hear your input and feedback. Your perspective is invaluable as we refine our plans for the upcoming year.

Let us know what you think—your ideas can make a big difference! 🎉

**Comment by LinPr** [comment]
cool🤗

**Comment by camilamacedo86** [comment]
The roadmaps are in: https://github.com/kubernetes-sigs/kubebuilder/tree/master/roadmap
We try to keep updated
If you have any suggestions, please feel free to raise a PR
